### PR TITLE
835314-Adding-API-links-in-SfNumericEntry-UG-documentation

### PR DESCRIPTION
**Pull request by AarthiArjunan**
Description:
Here I have added the volume 2 release controls Migration Page link in the common folder.

### CODE REVIEW

File: MAUI/Common/migration.md [hunk 1]
```diff
@@ -446,7 +446,7 @@ Currently working on delivering brand-new .NET MAUI controls that can be used in
 			<a href="https://help.syncfusion.com/xamarin/expander/getting-started">SfExpander</a><br/>
 		</td>
 		<td rowspan="1" valign="top">
-			Underway<br/>
+			<a href="https://help.syncfusion.com/maui/expander/migration">SfExpander</a><br/>
```
Comment (Choza-rajan) [inline MAUI/Common/migration.md:449]: Move this to top

File: MAUI/Common/migration.md [hunk 2]
```diff
@@ -470,7 +470,7 @@ Currently working on delivering brand-new .NET MAUI controls that can be used in
 			<a href="https://help.syncfusion.com/xamarin/numericupdown/getting-started">SfNumericUpDown</a><br/>
 		</td>
 		<td rowspan="1" valign="top">
-			Underway<br/>
+			<a https://help.syncfusion.com/maui/numericentry/migration">SfNumericEntry</a><br/>
```
Comment (Choza-rajan) [inline MAUI/Common/migration.md:473]: href is missing

Comment (AarthiArjunan) [reply MAUI/Common/migration.md]: Addressed the changes.

File: MAUI/Common/migration.md (hunk 1 above)
Comment (AarthiArjunan) [reply MAUI/Common/migration.md]: Addressed the changes.

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**